metrics/prometheus/prometheus.go

```diff
@@ -0,0 +1,272 @@
+package metrics
+
+import (
+	"github.com/prebid/prebid-cache/config"
+	"github.com/prometheus/client_golang/prometheus"
+	"time"
+)
+
+/**************************************************
+ * Constants and global variables
+ **************************************************/
+var TenSeconds time.Duration = time.Second * 10
+var AddLabel string = "add"
+var ErrorLabel string = "error"
+var BadRequestLabel string = "bad_request"
+var JsonLabel string = "json"
+var XmlLabel string = "xml"
+var DefinesTTLLabel string = "defines_ttl"
+var InvFormatLabel string = "invalid_format"
+var SubstractLabel string = "substract"
+var CloseLabel string = "close"
+var AcceptLabel string = "accept"
+
+/**************************************************
+ *	Object definition
+ **************************************************/
+type PrometheusMetrics struct {
+	Registry    *prometheus.Registry
+	Puts        *PrometheusRequestStatusMetric
+	Gets        *PrometheusRequestStatusMetric
+	PutsBackend *PrometheusRequestStatusMetricByFormat
+	GetsBackend *PrometheusRequestStatusMetric
+	Connections *PrometheusConnectionMetrics
+	ExtraTTL    *PrometheusExtraTTLMetrics
+}
+
+type PrometheusRequestStatusMetric struct {
+	Duration      prometheus.Histogram   //Non vector
+	RequestStatus *prometheus.CounterVec // CounterVec "status": "add", "error", or "bad_request"
```

Comment:
Does 'add' indicate success? I find that status string a bit confusing.

Reply:
Label string has been modified